src/mappings/Rewards.ts

```diff
@@ -206,20 +206,26 @@ async function handleSlashForTxHistory(slashEvent: SubstrateEvent): Promise<void
         return;
     }
 
-    const currentEra = (await api.query.staking.currentEra()).unwrap()
-    const slashDeferDuration = api.consts.staking.slashDeferDuration
-
-    const slashEra = slashDeferDuration == undefined 
-    ? currentEra.toNumber()
-    : currentEra.toNumber() - slashDeferDuration.toNumber()
-
-    const eraStakersInSlashEra = await api.query.staking.erasStakersClipped.entries(slashEra);
-    const validatorsInSlashEra = eraStakersInSlashEra.map(([key, exposure]) => {
-        let [, validatorId] = key.args
-
-        return validatorId.toString()
-    })
-    const validatorsSet = new Set(validatorsInSlashEra)
+    const eraWrapped = await api.query.staking.currentEra()
+    let validatorsSet = new Set()
+    let slashEra
+
+    if (eraWrapped.unwrap) {
+        const currentEra = eraWrapped.unwrap()
+        const slashDeferDuration = api.consts.staking.slashDeferDuration
+    
+        slashEra = slashDeferDuration == undefined 
+            ? currentEra.toNumber()
+            : currentEra.toNumber() - slashDeferDuration.toNumber()
+
+        const eraStakersInSlashEra = await api.query.staking.erasStakersClipped.entries(slashEra);
+        const validatorsInSlashEra = eraStakersInSlashEra.map(([key, exposure]) => {
+            let [, validatorId] = key.args
+
+            return validatorId.toString()
+        })
+        validatorsSet = new Set(validatorsInSlashEra)
+    }
 
     const initialValidator: string = ""
```

Comment:
Looks like we will have slash validator as empty string. Let's refactor and pass null as initial validator